src/clr-angular/button/button-loading/loading-button.ts

```diff
@@ -73,12 +73,14 @@ export class ClrLoadingButton implements LoadingListener {
     switch (state) {
       case ClrLoadingState.DEFAULT:
         this.renderer.removeStyle(this.el.nativeElement, 'width');
+        this.renderer.removeStyle(this.el.nativeElement, 'transform');
         if (!this.disabled) {
           this.renderer.removeAttribute(this.el.nativeElement, 'disabled');
         }
         break;
       case ClrLoadingState.LOADING:
         this.setExplicitButtonWidth();
+        this.renderer.setStyle(this.el.nativeElement, 'transform', 'translatez(0)');
```

Comment:
I just realized something after approving, do you mind adding a comment explaining briefly why we need this, maybe adding a link to this PR or the original issue? Thanks.